### PR TITLE
usbip: add page

### PR DESCRIPTION
**Pull request by Managor**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
usbip-utils 2.0

### CODE REVIEW

File: pages/linux/usbip.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# usbip
+
+> Tool for using USB devices remotely.
+> More information: <https://usbip.sourceforge.net/>.
+
+- List all local USB devices and their bus ID's:
+
+`usbip list --local`
+
+- Start a usbip daemon on the server:
```
Comment (kbdharun) [inline pages/linux/usbip.md:10]:
```suggestion
- Start a `usbip` daemon on the server:
```

File: pages/linux/usbip.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# usbip
+
+> Tool for using USB devices remotely.
+> More information: <https://usbip.sourceforge.net/>.
```
Comment (kbdharun) [inline pages/linux/usbip.md:4]:
```suggestion
> More information: <https://usbip.sourceforge.net>.
```
`/` can be removed here as the default URL doesn't use it.

File: pages/linux/usbip.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- List all local USB devices and their bus ID's:
+
+`usbip list --local`
+
+- Start a usbip daemon on the server:
+
+`systemctl start usbipd`
+
+- Bind an USB device to usbip on the server:
```
Comment (kbdharun) [inline pages/linux/usbip.md:14]:
```suggestion
- Bind a USB device to `usbip` on the server:
```

File: pages/linux/usbip.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+`sudo usbip bind --busid={{bus_id}}`
+
+- Load the kernel module required by usbip on the client:
```
Comment (kbdharun) [inline pages/linux/usbip.md:18]:
```suggestion
- Load the kernel module required by `usbip` on the client:
```

File: pages/linux/usbip.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+`sudo modprobe vhci-hcd`
+
+- Attach to the usbip device on the client (bus ID is the same as on the server):
```
Comment (kbdharun) [inline pages/linux/usbip.md:22]:
```suggestion
- Attach to the `usbip` device on the client (bus ID is the same as on the server):
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]

File: pages/linux/usbip.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
+# usbip
+
+> Tool for using USB devices remotely.
```
Comment (acuteenvy) [inline pages/linux/usbip.md:3]:
```suggestion
> Use USB devices remotely.
```

**Review by acuteenvy — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**